setup.py

```diff
@@ -45,7 +45,8 @@
     include_package_data=True,
     install_requires=[
         "pytket ~= 1.21",
-        "qiskit ~= 0.44.0",
+        "qiskit ~= 0.44.3",
```

Comment:
Do you want to put this into the changelog?

Reply:
Yes, I forgot to commit it...

Reply:
Done.